Source/WebCore/css/BasicShapeFunctions.cpp

```diff
@@ -80,12 +83,39 @@ static SVGPathByteStream copySVGPathByteStream(const SVGPathByteStream& source,
 Ref<CSSValue> valueForBasicShape(const RenderStyle& style, const BasicShape& basicShape, SVGPathConversion conversion)
 {
     auto createValue = [&](const Length& length) {
-        return CSSPrimitiveValue::create(length, style);
+        return CSSPrimitiveValue::create(length.isAuto() ? Length(0, LengthType::Percent) : length, style);
```

Comment:
From https://drafts.csswg.org/css-shapes-1/#basic-shape-computed-values :

> An auto value makes the edge of the box coincide with the corresponding edge of the [reference box](https://drafts.csswg.org/css-shapes-1/#basic-shape-reference-box): it’s equivalent to 0% as the first (top) or fourth (left) value, and equivalent to 100% as the second (right) or third (bottom) value.

Is this conversion correct? Maybe converting to 100% in the relevant cases would fix the interpolation tests?

Reply:
It’s correct. There are no `100` cases, because we are always converting to `inset` and so it’s always `0` (near the edge). Earlier ways I wrote the code had some `100` cases, but refactoring led to it just being this. I’ve looked at remaining failing interpolation tests and the issues are the interpolation not the values.

Reply:
Oh, actually, I thought of another way to explain the refactoring. Rather than have a separate case of `100` in here, I put a special case for `auto` into `createReflectedValue`. If I had not done that, then I would have had to pass in a value of `0` or `100`, and it would have computed `100% - 100%` and ended up `0%` anyway.